src/ProjectWindow.cpp

```diff
@@ -1839,6 +1852,27 @@ void ProjectWindow::DoZoomFit()
    window.TP_ScrollWindow(start);
 }
 
+void ProjectWindow::ShowEffectsPanel(Track* track)
+{
+
+   if(track == nullptr)
+      return;
+
+   if(!mEffectsWindow->IsShown())
+   {
+      //Restore previous effects window size
+      auto effectsWindowSize = mEffectsWindow->GetSize();
+      mEffectsWindow->Show();
+      mContainerWindow->SplitVertically(mEffectsWindow, mTrackListWindow, effectsWindowSize.GetWidth());
+   }
+}
+
+void ProjectWindow::HideEffectsPanel()
+{
+   mEffectsWindow->Hide();
+   mContainerWindow->Unsplit(mEffectsWindow);
```

Comment:
Is Unsplitting enough without the call to Hide?

Reply:
Turned out - yes, thanks